### ISSUE TITLE
Can I Have Four Rating Level instead of Five?

### ISSUE THREAD

**Issue by ahikmatf** [sujithkanna/SmileyRating#27]
I need to remove the "Okay" rating level, so the user can only rate either "positive" or "negative" feedback. I am wondering how to achieve this kind of workaround.

Thank you.

**Comment by ghost** [comment]
+1 I wonder if it's possible to set even one smiley for "aggregation" purposes of total feedback. Thanks.

**Comment by sujithkanna** [comment]
Currently, that feature is not made. But we are planning to make it in near future.

**Comment by MuhammadFaisalIchal** [comment]
So, decreasing the smiley icon (like only 3 icon) is impossible?

**Comment by addisonElliott** [comment]
Yes, that feature is not possible at the moment. A lot of the open issues in this repository are about making the library 'more flexible', or more configurable. The library needs a bit of reworking to add this ability since currently it just draws the 5 smileys.

**Comment by MuhammadFaisalIchal** [comment]
Well thanks addison.
I'm waiting for the new library. Actually i hoped can help too if 'm able

**Comment by addisonElliott** [comment]
No problem. Of course you can help, that's the beauty of open-source. Go ahead and fork/clone this repository and try experimenting. 

If you get to a point that you have to stop or cannot finish, I recommend creating a pull request (PR) so that others can potentially build off what you have done.

**Comment by MuhammadFaisalIchal** [comment]
Ok. i'll try

**Comment by princenaman** [comment]
+1

**Comment by Sammy03** [comment]
Any updates on this. This is a must have

**Comment by hasan5151** [comment]
still no update?

**Comment by maohieng** [comment]
I created a pull request for this question.
See [pull/53](https://github.com/sujithkanna/SmileyRating/pull/53)
